plugins/dev-tools/src/field_test_helpers.mocha.js

```diff
@@ -61,7 +59,9 @@ export function assertFieldValue(field, expectedValue,
     expectedText = undefined) {
   const actualValue = field.getValue();
   const actualText = field.getText();
-  expectedText = expectedText || String(expectedValue);
+  if (expectedText === undefined) {
```

Comment:
would you want to do the same if expectedText is null

Reply:
I'm not sure if we would ever intentionally check that expectedText, but leaving it as this allows us to do so. My intention is just to check whether expectedText was passed.